group_vars/shxco_qa/vars.yml

```diff
@@ -0,0 +1,16 @@
+###
+# QA specific variables for mep-django
+###
+---
+
+# allow any host, since we're behind PUL load balancer
+allowed_hosts:
+    - "*"
+# email prefix for dev account emails
+email_prefix: '[QA S&co] '
+
+# solr settings
+# TODO: move somewhere common?
+zk_host: "lib-zk-staging1:2181,lib-zk-staging2:2181,lib-zk-staging3:2181/solr8"
+solr_url: "http://lib-solr8-staging.princeton.edu:8983/solr/"
+solr_server: "{{ groups['solr_staging'][0] }}"
```

Comment:
is it not safe to put these in `/group_vars/all`? they're only for the new solr role, right?

Reply:
`zk_host` is only used for the new solr role, but solr url and solr server are used for the old deploys

Reply:
was thinking to hold off moving anywhere until we have the migrations done, and then we can clean up a ton of stuff from the old deploys and old variables. (in spite of the TODO comment)